content/learn/book/welcome/_index.md

```diff
@@ -1,9 +1,48 @@
 +++
-title = "Welcome to Bevy"
+title = "Welcome"
 weight = 1
 sort_by = "weight"
 template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- **Rust-first:** all the safety, tooling and power that come with a modern systems language
+- **Completely free:** [free-of-charge](https://github.com/sponsors/cart), [open source](https://github.com/bevyengine/bevy/blob/main/LICENSE), and hackable with an architecture designed for modularity
+- **Innovative ECS:** powered by an ergonomic, full-native entity-component-system architecture that makes writing clear, performant gameplay and engine logic a joy
+- **Cross-platform:** support Windows, MacOS, Linux, web*, Android* and iOS with a single code base
+- **Effortless parallelism:** a data-oriented design and automatic parallel scheduler provide a high-performance foundation for your game
+
+This book is designed for new and experienced users looking for a thoughtful, [explanation-first](https://diataxis.fr/explanation/) guide to the engine's essential features.
+If you just want to try it out hands-on, jump into one of our [quick-start guides](TODO: add link) instead!
+
+*: support for these platforms is still limited; you may encounter some missing features or a more involved setup process
+
+## Stability warning
+
+Bevy aims to be a general-purpose game engine capable of handling arbitrary 2D or 3D games.
+However, Bevy is still in its infancy, and there are very good reasons not to choose it for your next project.
+No mature solutions currently exist for:
+
+- animation
+- networking
+- console support
+
+The following areas are part of the engine itself, but are very immature:
+
+- audio
+- user interfaces
+
+Solid third-party solutions currently exist for:
+
+- [advanced audio](https://crates.io/crates/bevy_kira_audio)
+- [realistic physics](https://github.com/dimforge/bevy_rapier)
+
+While Bevy's modular architecture makes it relatively easy to integrate your own (or third-party) solutions, be mindful that Bevy does not provide a complete game solution out of the box *yet*.
+[Contributions to the engine](https://github.com/bevyengine/bevy/) are extremely welcome, but if you want something fully-featured to make your dream game *today*, check out [Godot](https://godotengine.org/) instead!
+
+Unsurprisingly, Bevy's rapid pace of development means that our APIs will break, repeatedly, in both deep and pervasive ways.
```

Comment:
One thing to consider: Rust is often considered to be in line with "if it compiles, it works" - however, I think bevy is not matured there yet. Maybe a good to mention in docs?

(920+ unwrap() or panic!'s in code currently - I've personally been caught by a surprise few times to runtime crashes)

Reply:
That's probably a good caveat. Compared to other game engines, it's surprisingly "if it compiles it works". But much less so for people coming to games from the Rust ecosystem!

Reply:
I think "if it compiles, it works" is an unrealistic goal because Rust's static guarantees are just a subset of its capabilities. I also don't think that "number of unwraps" is a particularly good measure of code maturity. There are plenty of cases where the only real option is an internal unwrap.

That being said, except in a few specific cases, Bevy internals shouldn't crash from normal "userspace" code (and they do currently in a number of cases).

Its worth calling out that Bevy currently has more rough edges / bugs / crashes than a stable product would. However this page is also the first thing Bevy users will see and I consider it a key part of "Bevy marketing". We should do it in a way that doesn't (1) scare people away or (2) give the wrong impression about our goals and priorities. Feel free to add a line here, but (as I've already warned @alice-i-cecile) I have a lot of very specific goals for this page and I'll probably follow this pr up with a lot of changes.

Reply:
> except in a few specific cases

Are these cases outlined elsewhere? While I agree that they shouldn't be listed on the first welcome page, perhaps a link to the "rough edges" would be appropriate.

Reply:
There was good discussion in Discord about the unwrap's as well yesterday. I agree that there will be unwraps, and some of them are totally okay as well (e.g. when starting app, during development work). Also, of the 920+ quite many seemed to be in the tests code, which I did not consider when grepping.

Good point about the marketing, maybe the extra warning should not be yet - the section header "Stability warning" already gives a hint about it. 

Also, over time the runtime panic's will get lower and lower. However, I would still be very mindful of this, since an unanticipated runtime panic at wrong point (e.g. demo time, or in production) will quickly turn new adopters away.

Reply:
> Are these cases outlined elsewhere? While I agree that they shouldn't be listed on the first welcome page, perhaps a link to the "rough edges" would be appropriate.

They are not. It will be hard to maintain the list unless it integrates directly with the issue tracker. Maybe we should have a label for "UX speedbumps". We can link to that specific label. And that way whenever we resolve an issue, the list is automatically updated.